src/style.css

```diff
@@ -1 +1,118 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.ttf);
+}
```

Comment:
You can just import fonts from google-fonts, no need to use @font-face and upload font files to PR